source/blog/2017-09-27-webadmin-lldp.html.md

```diff
@@ -0,0 +1,30 @@
+---
+title: LLDP information in webadmin
+author: amusil
+tags: webadmin, networking
+date: 2017-09-27 09:00:00 CET
+---
+
+In the oVirt 4.2.0 we have introduced support for LLDP protocol. 
+LLDP stands for Link Layer Discovery Protocol, which is protocol used by network devices for advertising identity and capabilities to neighbors on a LAN. [More information about LLDP](https://learningnetwork.cisco.com/docs/DOC-26851).
+
+READMORE
+
+### Useful information
+
+The information gathered by protocol can be used for better network configuration. UI shows most important informations e.g. System name, Port description and VLAN IDs. Administrator can set VLAN ID according to switch configuration. In case of some problem Port description can become handy as it shows switch port number to which is network interface connected.
```

Comment:
Please use shorter lines (80 columns is for old people, but I'm old). it is hard to review long lines here.

"gathered by protocol" -> gathered by *the* protocol

"informations" -> pieces of information

Reply:
Please drop the whole sentence "Administrator can set VLAN ID according to switch configuration. In case of some problem Port description can become handy as it shows switch port number to which is network interface connected." and rewrite it as a user-story paragraph. The "story" should be well connected to the screenshot. I'm expecting something in the lines of 

When adding a host to an oVirt cluster, the administer usually need to attach various networks to it. However,modern hosts have multiple interfaces each with its own non-descriptive name. How would the administrator know whether she needs to attach a network named `m2` with VLAN_ID 162 to `ens2f0` or to ens2f1 ? With ovirt-4.2, she can hover over ens2f0 and see to which switch is this NIC connected to, and whether the peer switch supports vlan 162.

Similarly, when creating a bond of mode 4 (LACP) it is highly important to understand to which port is each NIC connected to. For example, when creating `bond0` in the image above, it is important to pick the two NICs that are connected to to ports on the same peer switch, and that these to ports are in the same `port group`. Quite often, administrators choose the wrong NICs, and end up with badly-configured bonds. With LLDP information readily available, the admin can easily choose those two NICs based on their peer switch and port name.

You are most welcome to write the above text in your own words. My point is explain why this feature is so important.